### ISSUE TITLE
Inner Hits on nested query

### ISSUE THREAD

**Issue by Surt** [babenkoivan/elastic-scout-driver-plus#32]
Hi!

I'm triying to build complex queries with the system but I notice lack of options. I tried composing query builders etc but I always end up with some limitation.
Don't get me wrong, what I think is that to map EVERY elasticsearch query option and it's api changes in time, should be inneficient and impossible on the long run for the library.

For long and complex programmatic searchs it is easy to use the rawSearch and compose it as arrays. Also, usually it is easier to read those arrays than the resulted ones using the library interface. 

At this moment the problem was with inner-hits on a nested query https://www.elastic.co/guide/en/elasticsearch/reference/6.8/search-request-inner-hits.html#nested-inner-hits

maybe you end up with a solution that I didn't see?

**Comment by babenkoivan** [comment]
Hey @Surt, I'm trying to keep balance between `easy to use` and `flexible`. This library is limited indeed, the most flexible solution for ES in PHP is https://github.com/elastic/elasticsearch-php, which I'm using under the hood.

I guess the only option to get nested hits at the moment is to use `raw`:

```php
$rawSearchResult = MyModel::rawSearch()->query(/****/)->raw();
```

**Comment by Surt** [comment]
Yes, I need the raw results I was refering to the nested Query Builder wich lacks inner-hits.

Now, with the results I need the raw as you said but... wouldn't be great to have **the result hit mapped on the `match` model**? I see that the document is mapped, but it would be great to have the entire result `hit`.

With that functionality of "hit data" on the match I don't need to transverse the entire raw result and query for models, etc. ( I did already on previous versions of my app but It would be great to delete that overhead.

**Comment by babenkoivan** [comment]
@Surt, it makes sense. I've created a note to implement `Match::raw()` or `Match::hit()`.

**Comment by Surt** [comment]
> @Surt, it makes sense. I've created a note to implement `Match::raw()` or `Match::hit()`.

That woud be INCREDIBLE! as always. thx @babenkoivan

**Comment by babenkoivan** [comment]
@Surt, `Match::raw()` is now available in [v1.14.0](https://github.com/babenkoivan/elastic-scout-driver-plus/releases/tag/v1.14.0) 🎉

**Comment by Surt** [comment]
!!!! eager to try it! Will update here!
